nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -88,6 +88,12 @@ $(CsWinRTFilters)
     <Message Text="$(CsWinRTCommand)" Importance="$(CsWinRTMessageImportance)" />
     <Exec Command="$(CsWinRTCommand)" />
   </Target>
+
+  <Target Name="CsWinRTRemoveShimDll" Condition="'$(CsWinRTComponent)'!='true'">

```

Comment:
per discussion, we should stage the work:

1. add winrt.host.dlls, winrt.host.shim.dll and disable all references to these, since no project that references the cswinrt nuget will ever directly consume them.  check this is in to unblock partners to manually stage, per Adam's authoring sample.
2. add component-specific props/targets files that:
  a. are ignored by non-component projects
  b. are bundled into component project packages (this may require some manual nuspec entries)
  c. provide logic for staging arch-specific winrt.host.dll and winrt.host.shim.dll for native referencing projects

Any of the logic in (2) above that cannot be fully automated should be documented in the repo and brief mention made in the nupkg readme.txt (e.g., "for information on creating a C# WinRT component package, see this link...")

Reply:
Thanks! I copied and posted this on the original issue we have been using as well.